src/core/JsonConfigCommandCenter.ts

```diff
@@ -14,18 +18,115 @@ export class JsonConfigCommandCenter{
     }
 
     public static get editor(): CogniteJsonEditor | null {
-        if(JsonConfigCommandCenter.editorInstance) {
+        if (JsonConfigCommandCenter.editorInstance) {
             return JsonConfigCommandCenter.editorInstance;
         } else {
             console.error("Cannot retrieve editor before instantiation!");
             return null;
         }
     }
 
+    public static get currentJson(): any {
+        const currentJsonText = JsonConfigCommandCenter.editor?.getText();
+        if (currentJsonText) {
+            return JSON.parse(currentJsonText);
+        }
+    }
+
+    public static loadDigitalTwins = (digitalTwinConfigMap: (twinNames: Map<number, unknown> | null) => void): void => {
+        (async () => {
+            await cogniteClient.get(`${cogniteClient.getBaseUrl()}/api/playground/projects/${cogniteClient.project}/twins`)
+                .then(response => {
+                    console.log("Retrieved Digital Twin List successfully");
+                    const twins = response.data.data?.items;
+                    const twinMap = new Map();
+
+                    for (const twin of twins) {
+                        const twinId = twin.id;
+                        twinMap.set(twinId, twin);
+                    }
+                    digitalTwinConfigMap(twinMap);
+                })
+                .catch(error => {
+                    console.error("Retrieved Digital Twin List failed");
+                    JsonConfigCommandCenter.errorAlert("Save Cancelled!", error);
+                });
+        })();
+    }
+
+    // call with undefind values to create new config
+    public static onJsonConfigSelect = (
```

Comment:
i feel this is a front end logic no need to put this in this class. since it's just marking selected config in the list. keep them in the component. Use this class only to interact with editor instance from other panels.